### PR TITLE
Add ASN.1 module

### PR DESCRIPTION
**Pull request by richsalz**
Move RFC Editor note to Abstract.

### CODE REVIEW

**Comment by richsalz** [comment]
Add Russ's _real_ draft ASN.1 module :)

File: draft-ietf-trans-rfc6962-bis.md [hunk 1]
```diff
@@ -156,6 +156,11 @@ used to send various CT log artifacts.
 Logs are network services that implement the protocol operations for submissions
 and queries that are defined in this document.
 
+\[RFC Editor: please update 'RFCXXXX' to refer to this document,
+once its RFC number is known, through the document. Note that the use
+in the ASN.1 Appendix "RFCRFCXXXX" is deliberate. \]
```
Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md:161]: I think `Note that the use in the ASN.1 Appendix "RFCRFCXXXX" is deliberate` was applicable to your first draft, but is no longer applicable.  If you agree, please remove this sentence.

File: draft-ietf-trans-rfc6962-bis.md [hunk 2]
```diff
@@ -2391,3 +2393,87 @@ SCTs:
...
+   CRITICALITY { FALSE } }
+
+id-pkix-ocsp-transparencyInfo OBJECT IDENTIFIER ::=
+   id-ce-transparencyInfo
+
+--
+-- Section 8.1.2.  Reconstructing the TBSCertificate
+--
+
+ext-embeddedSCT EXTENSION ::= {
```
Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md:2467]: To make it absolutely clear which CT version this extension belongs to, WDYT about naming this extension `ext-embeddedSCT-CTv1` and the corresponding OID `id-ce-embeddedSCT-CTv1`?

**Comment by richsalz** [comment]
took both your suggestions and pushed a new commit.

**Pull request merged**